src/Components/AddToDo.jsx

```diff
@@ -30,14 +36,18 @@ const AddToDo = () => {
           <input type="submit" value="ADD" onClick={add}></input>
         
         </span>
+        <span>
+          <input type="submit" value="Clear" onClick={clearTodo}></input>
+        
+        </span>
       </div>
       <div className="list">
-        {list.map((element, index) => {
-          return <div className="items" key={index}>{element}</div>;
+        {myProp.todoArray.map((element) => {
```

Comment:
what if `myProp.todoArray` is undefined? in that case it will crash

Reply:
It's Done. Applied Check on this